CHANGELOG.md

```diff
@@ -11,8 +27,41 @@
     features = ["3.10"]
     ```
 
+- Signal handlers [take the receiver by reference][gtk226-1] and support [more
+  specific receiver types][gtk226-2].
+
+- New [safe event bindings][gdk95] are [passed to signal handlers][gtk244].
+  These bindings don't have many getters and setters implemented yet but the low
+  level event structs are still reachable [via `as_ref`][examples68].
+
+- New safe [`Error`][glib86-error] and [`Value`][glib95] bindings. `ListStore` and
+  `TreeStore` [changed accordingly][gtk238], traded `set_string` for more
+  powerful [`set`][gtk241].
+
+- In an ongoing move to [reflect integer unsignedness properly][gtk246]
+  `Notebook` and partly `ListStore` and `TreeStore` switched from `i32` to
```

Comment:
This sentence seems a bit weird. What about "Notebook and partly both ListStore and TreeStore"? (yes, this __both__ changes everything :p )